client/js/components/show-hide.js

```diff
@@ -11,8 +11,24 @@ const showHide = (state) => {
   const init = () => {
     el.setAttribute('aria-haspopup', 'true');
     options.trigger.setAttribute('aria-expanded', getActive());
+
+    setDataTrackAria(getActive());
   };
 
+  function setDataTrackAria(value) {
+    const dataTrackOptions = options.trigger.getAttribute('data-track-click');
+
+    if (!dataTrackOptions) return;
+
+    const updatedTrackOptions = JSON.stringify(Object.assign(
+      {},
+      JSON.parse(dataTrackOptions),
+      {actionFromAriaExpanded: value ? 'open' : 'close'}
```

Comment:
Open and close can be confusing, has it been set to open, or set to closed, or is open, or is closed.
We could make it more explicit by setting `wasOpened` or `isOpen`?

I prefer the later as it records the action.